target/linux/generic/patches-4.4/308-mips32r2_tune.patch

```diff
@@ -1,17 +0,0 @@
-Add -mtune=34kc to MIPS CFLAGS when building for mips32r2
```

Comment:
-mips32r2 is okay, but your patch also does an -mtune=34kc, this will not work on my 4kc ...

I add this flag to CONFIG_TARGET_OPTIMIZATION: -march=4kec -EB ... this works.